code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -65,14 +65,14 @@
 	if(stage == 4)
 		for(var/datum/surgery/S in owner.surgeries)
 			if(S.location == "chest" && S.organ_to_manipulate.open >= ORGAN_ORGANIC_OPEN)
-				AttemptGrow(burst_on_success = FALSE) ///If you managed to get this far, you deserve to be rewarded somewhat
+				alien_embryo_grow(burst_on_success = FALSE) ///If you managed to get this far, you deserve to be rewarded somewhat
 				return
-		AttemptGrow()
+		alien_embryo_grow()
 
 
 
-/obj/item/organ/internal/body_egg/alien_embryo/proc/AttemptGrow(burst_on_success = TRUE)
-	if(!owner || polling)
+/obj/item/organ/internal/body_egg/alien_embryo/proc/alien_embryo_grow(burst_on_success = TRUE)
+	if(QDELETED(owner) || polling)
```

Comment:
This won't fix the runtime. The runtime happens due to the sleep at line 79. In that time the `owner` could have been nulled. It shouldn't ever be not null but QDELETED, else organ code fucked up somewhere.

Reply:
wait yeah I'm stupid